src/msgs/OrderInfo.proto

```diff
@@ -92,6 +93,19 @@ message OrderInfo {
   repeated Order orders = 1;
 }
 
+message UnconfirmedDelivery {
```

Comment:
An optional field in OrderInfo would probably suffice.

Reply:
See discussion above.